install/kubernetes/istio-auth.yaml

```diff
@@ -94,6 +109,20 @@ roleRef:
   name: istio-pilot-istio-system
   apiGroup: rbac.authorization.k8s.io
 ---
+# Grant permissions to the Sidecar initializer
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: istio-sidecar-initializer-admin-role-binding-istio-system
+subjects:
+- kind: ServiceAccount
+  name: istio-sidecar-initializer-service-account
```

Comment:
Remove sidecar from the name

Reply:
The naming was derived from kelsey heightower's envoy-initializer, i.e. s/envoy/istio-sidecar. I can change it if its confusing, but I should start with istio/pilot first. Also, any chance istio will use other initializers?

Reply:
We never know what future holds in store, there may be other initializers. The name is still confusing. We call the proxy a side car proxy, so I expect a sidecar initializer to be a sidecar...

How about istio-proxy-initializer or istio-proxy-injector?